src/main/java/pl/ug/virtualofficebackend/domain/user/internal/UserServiceImpl.java

```diff
@@ -0,0 +1,13 @@
+package pl.ug.virtualofficebackend.domain.user.internal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.ug.virtualofficebackend.domain.user.boundary.UserService;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    @Autowired
+    private UserRepository userRepository;
```

Comment:
Podobno bezpieczniej uzywac Autowired na konstruktorach

Reply:
Ok, to zaraz poprawię